istio-proxy/istio-proxy.spec

```diff
@@ -27,7 +27,7 @@
 %global _prefix /usr/local
 
 Name:           istio-proxy
-Version:        2.0.0
+Version:        2.0.1
```

Comment:
This is missing a bump in commit sha, no?

Reply:
aargh, good catch